src/contract-testing/test-helpers/mock-server.ts

```diff
@@ -0,0 +1,84 @@
+import express from 'express';
+import { handle } from '../../handlers/status-retriever-handler';
+import { ContextExamples } from '@aws-lambda-powertools/commons';
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from 'aws-lambda';
+
+let result: APIGatewayProxyResult;
+let server: any;
```

Comment:
If we can't figure out what type server should be, could we please turn off the unexpected any rule for this line please? :) so we avoid the pop up below. I tried many different types, but none seemed to work